after/ftplugin/xml.vim

```diff
@@ -1,12 +1,6 @@
 if gradle#isGradleProject()
-
-  call gradle#setClassPath()
```

Comment:
For now revert back the deletes. It takes time because it is actually running gradle to detect if the project includes the android plugin and in the way finds out the target sdk version.

My plan is to implement this using neovim async or vimproc so it is done in the background. If everything works as advertised then there will be no lag at all in the future and we still get accurate android project detection. The previous heuristic based on parsing AndroidManifest.xml was failing for lots of cases.

The idea of using javacomplete2 may work but depending on a plugin that not everybody may use is not appealing. I personally don't use it so would be useless for me. If this is to be merged it must work as alternative and not as only way to get classpath information.